packages/rich-text-from-markdown/src/index.ts

```diff
@@ -117,6 +132,33 @@ const buildGenericBlockOrInline = async (
   ];
 };
 
+const buildTableCell = async (
+  node: MarkdownNode,
+  fallback: FallbackResolver,
+  appliedMarksTypes: string[],
+): Promise<Array<Block>> => {
+  const content = await mdToRichTextNodes(node.children, fallback, appliedMarksTypes);
+
+  /**
+   * We should only support texts inside table cells.
+   * Some markdowns might contain html inside tables such as <ul>, <blockquote>, etc
+   * but they are pretty much filtered out by markdownNodeTypes and nodeContainerTypes variables.
+   * so we ended up receiving only `text` nodes.
+   * We can't have table cells with text nodes directly, we must wrap text nodes inside paragraphs.
+   */
+  return [
+    {
+      nodeType: BLOCKS.TABLE_CELL,
+      content: content.map(contentNode => ({
```

Comment:
Could there be a situation where `content` would already consist of paragraphs? We can't have nested paragraphs.

Also, do we handle the case where there's e.g. a list inside a table, which we don't currently support? In this case, there should probably only be plain text or some of the raw markdown be inserted into the table :thinking:

Reply:
I don't think so, from all the tests I've performed, the markdown parser always returns `tableCell` -> `text`. Talking to Chris, we can't have text as direct child of a table cell so we wrap them with paragraphs.

Reply:
Yes, I think it's proven with your
```
| <div><p>Rewe</p> and <p>Edeka</p></div> | Germany |
```
test case :clap: